demo.yaml/mxcube-web/ui.yaml

```diff
@@ -1,5 +1,13 @@
 ---
 
+sample_list_view:
```

Comment:
Maybe just rename to `sample_list_view_modes` so it's really clear and consistent with the front-end?

I get that the page itself is called "Sample list", but the fact that the other page is called "Sample view" makes the term `sample_list_view` slightly confusing to me.